<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>287</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>CIDAS</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Segoe UI Emoji&quot;;
background-color: rgb(243, 243, 243);</string>
      </property>
      <property name="text">
       <string>VIEW CLIENT HISTORY</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Segoe UI Emoji&quot;;
background-color: rgb(208, 208, 208);</string>
      </property>
      <property name="text">
       <string>VIEW BRIEF HISTORY</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Segoe UI Emoji&quot;;
background-color: rgb(163, 163, 163);</string>
      </property>
      <property name="text">
       <string>VIEW REAL TIME DATA</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Segoe UI Emoji&quot;;
background-color: rgb(85, 85, 85);</string>
      </property>
      <property name="text">
       <string>VIEW CLIENT REPORT</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>332</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
